lib/knackhq/client/recursive_symbolize_keys.rb

```diff
@@ -0,0 +1,21 @@
+# Recursively symbolizes keys in hash
```

Comment:
We can use these methods on Arrays of Hashes directly rather than on Hashes and map them. Its the same operations but this time, we can use it on both Arrays and Hashes later on.